deploy/AllocatorInit.sol

```diff
@@ -138,6 +139,7 @@ struct AllocatorIlkConfig {
     address[] swapTokens;
     address[] depositTokens;
     address ilkRegistry;
+    address uniV3Factory;
```

Comment:
Since the factory address is the same on mainnet and goerli, we could consider just using a constant for it, both in here and in `AllocatorDeploy.sol`. Some newer deployments are using a [different address](https://docs.uniswap.org/contracts/v3/reference/deployments) though, so not feeling too strongly about it.

Reply:
If we want a constant then can just use it in the `Univ3Depositor` called itself.
In general passing the address allows for using it in local rpc testing and in mocks, so I think we can leave it as a parameter.